.circleci/config.yml

```diff
@@ -70,7 +74,22 @@ aliases:
   - &install_datadog
     run:
       name: Install Datadog
-      command: npm install -g @datadog/datadog-ci 
+      command: npm install -g @datadog/datadog-ci
+
+commands:
+  vitest-affected:
+    parameters:
+      mode:
+        description: "Test execution environment, incompatible tests will be ignored"
+        type: enum
+        enum: [ "node", "chrome" ]
+        default: "node"
+    steps:
+      - run:
+          command: pnpm nx affected --base=$NX_BASE --head=$NX_HEAD --target=vitest
+          environment:
+            VITEST_XML_REPORT: true
```

Comment:
<img width="1058" alt="image" src="https://github.com/dxos/dxos/assets/36420699/29d795b6-3c78-4ce2-afb4-1db996f68e5f">

Do we need an XML report?
If we do, could we also have inline results in the circle CI console?

Reply:
I think as the number of tests grow and console log ouput becomes bigger having an exported xml report is very useful. Added the second reporter in vitest.shared.ts, now it should print to console in addition to exporting the report